EMS/common-bundle/src/DependencyInjection/EMSCommonExtension.php

```diff
@@ -41,6 +41,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('ems_common.backend_url', $config['backend_url']);
         $container->setParameter('ems_common.backend_api_key', $config['backend_api_key']);
         $container->setParameter('ems_common.backend_api_verify', $config['backend_api_verify']);
+        $container->setParameter('ems_common.search_via_backend', $config['search_via_backend']);
```

Comment:
I do not like the naming of the parameter and configuration, in the service the variable is also called `useAdminProxy`

For me it should be under: EMS_ELASTICSEARCH_*

Proposal:   EMS_ELASTICSEARCH_PROXY_API